TinyInsights/ApplicationInsightsProvider.cs

```diff
@@ -309,4 +310,61 @@ public Task TrackDependencyAsync(string dependencyType, string dependencyName, s
         
         return Task.CompletedTask;
     }
+
+#region ILogger
+    public async void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception? exception, Func<TState, Exception?, string> formatter)
+    {
+        if(!IsEnabled(logLevel))
+        {
+            return;
+        }
+
+        var logTask = logLevel switch {
+            LogLevel.Trace => TrackPageViewAsync(GetEventName(eventId), GetLoggerData(logLevel, eventId, state, exception, formatter)),
+            LogLevel.Debug => Task.CompletedTask,
```

Comment:
is the Debug option correct ?

Reply:
Do you have any suggestion how I should log Debug data?

Reply:
I now use Debug.WriteLine to output data there. What do you think about that?